### PR TITLE
add login page and its component

### PR DESCRIPTION
**Pull request by KKDoddy**
### What does this PR do?
It adds the login page and the login basic component with necessary styles
### Description of Task to be completed?
- design a basic html css files for the login page
- design basic components for the login functionality
### How should this be manually tested?
clone the repo
install all packages through npm i
start the client UI through npm run start
### What are the relevant pivotal tracker stories?
N/A
### Screenshots
image

### CODE REVIEW

**Review by luc-tuyishime — COMMENTED** [review]
Why so many SVGs image for now?

File: src/components/login/Login.js [hunk 1]
```diff
@@ -0,0 +1,55 @@
+/* eslint-disable jsx-a11y/anchor-is-valid */
+import React, { Component } from 'react';
+import facebookLogo from '../../assets/images/social/facebook-logo.png';
+import googleLogo from '../../assets/images/social/google-logo.png';
+import '../../assets/css/style.css';
```
Comment (luc-tuyishime) [inline src/components/login/Login.js]: Import the file with sass Extention.

File: src/components/login/Login.js [hunk 2]
```diff
@@ -0,0 +1,55 @@
...
+            </div>
+
+            <a href="#" className="btn md-btn b-radius-circle bg-green text-center sm-title">Sign In</a>
+            <div className="text-center"><a href="#" className="link c-green">Forgot password?</a></div>
+          </div>
+        </form>
+        <div className="land-auth">
+          <div className="welcome-auth m-20 p-20 text-center">
+            <h1 className="lg-title m-bottom">
+Hello
```
Comment (luc-tuyishime) [inline src/components/login/Login.js]: format your code.

**Comment by KKDoddy** [comment]
@luc-tuyishime they are fontawesome icons, and we might need them later.

**Pull request closed without merging**